css/portfolio.css

```diff
@@ -0,0 +1,100 @@
+* {
+	box-sizing: border-box;
+/*	border: 1px solid red;*/
```

Comment:
It's strange to leave commented code like this in a pull request because the next person to look at the code might not know why it is commented or what you were trying to achieve by commenting the code